### PR TITLE
Prevent Undefined String Exceptions

### PR DESCRIPTION
**Pull request by micahalcorn**
This solves two cases where undefined strings cause origin-js to throw uncaught exceptions. Both occur when the DApp is accessed without a wallet, and one causes an error message to appear intermittently in the UI. I'm not sure if these are the ideal places to handle the missing strings, but I suspect this will all change soon anyway.

### CODE REVIEW

**Comment by micahalcorn** [comment]
error

File: origin-js/src/contractInterface/users/v01_adapter.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ export default class V01_UsersAdapter {
    * @return {Promise<string|null>}
    * @private
    */
-  async _getIdentityIpfsHash(address) {
+  async _getIdentityIpfsHash(address = '') {
```
Comment (franckc) [inline origin-js/src/contractInterface/users/v01_adapter.js]: As opposed to setting a default value in this method, I would
1. Add a safeguard that returns null if !address. e.g.
if (!address) {
  return null
}

2. Add a similar safeguard in the async get(address) method above at line 84:
if (!address) {
  return false
}

Comment (micahalcorn) [reply origin-js/src/contractInterface/users/v01_adapter.js]: @franckc if you wouldn't mind, I'm mid-other-branch right now dealing with mobile. I'll come back to this a bit later if you haven't gotten to it. Thanks!

**Comment by franckc** [comment]
No problem ! Done...

**Pull request merged**